lib/widgets/dashboard/current_balance.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:e_wallet/models/user_model.dart';
+import 'package:e_wallet/providers/user_provider.dart';
+import 'package:e_wallet/util/custom_format_util.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class CurrentBalance extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    UserModel user = Provider.of<UserProvider>(context).userModel;
+
+    return Center(
+      child: new AspectRatio(
+        aspectRatio: 100 / 25,
+        child: new Container(
+          decoration: new BoxDecoration(
+            shape: BoxShape.rectangle,
+            color: Theme.of(context).colorScheme.primary,
```

Comment:
You could use Color() class if you want use any colors using hexcodes

Reply:
Nice to know.